.gitignore

```diff
@@ -12,4 +12,5 @@ venv/
 *.egg-info
 *.log
 .idea/
+.vscode/
```

Comment:
Actually, depending on what is in that folder, sharing these `workspace` configurations for vscode could be a good idea. Already do that for the table https://github.com/plotly/dash-table/blob/master/.vscode/settings.json